index.bs

```diff
@@ -828,7 +828,12 @@ sequence&lt;{{WebTransportHash}}&gt; |serverCertificateHashes|, run these steps.
 1. Let |request| be a new [=/request=] whose [=request/URL=] is |url|, [=request/client=] is
    |client|, [=request/policy container=] is |client|'s
    [=environment settings object/policy container=], [=request/destination=] is an empty string,
-   and [=request/origin=] is |origin|.
+   [=request/origin=] is |origin| and [=request/redirect mode=] is "error". 
+
+Note: redirects are intentionally not exposed to the application. In cross-origin contexts, this
+would reveal information that would normally be blocked by CORS. In same-origin contexts, it 
+would encourage users to abuse the handshake as a vector for passing information. 
```

Comment:
```suggestion
Note: Redirects are not followed. Network errors caused by redirection are intentionally indistinguishable from other network errors.  In cross-origin contexts, this would reveal information that would normally be blocked
by CORS. In same-origin contexts, it would encourage users to abuse the handshake as a vector for passing information. 
```